README.txt

```diff
@@ -0,0 +1,237 @@
+# Project description
+
+Cloudstate is a specification, protocol, and reference implementation for providing distributed state management patterns suitable for **Serverless** computing. 
+The current supported and envisioned patterns include:
+
+* **Event Sourcing**
+* **Conflict-Free Replicated Data Types (CRDTs)**
+* **Key-Value storage**
+* **P2P messaging**
+* **CQRS read side projections**
+
+Cloudstate is polyglot, which means that services can be written in any language that supports gRPC, 
+and with language specific libraries provided that allow idiomatic use of the patterns in each language. 
+Cloudstate can be used either by itself, in combination with a Service Mesh, 
+or it is in envisioned that it will be integrated with other Serverless technologies such as [Knative](https://knative.dev/).
+
+Read more about the design, architecture, techniques, and technologies behind Cloudstate in [this section in the documentation](https://github.com/cloudstateio/cloudstate/blob/master/README.md#enter-cloudstate). 
+
+Cloudstate Python user language support is a library that implements the Cloudstate protocol and offers an elegant Python API 
+for writing entities that implement the types supported by the Cloudstate protocol.
+
+Cloudstate documentation can be found [here](https://cloudstate.io/docs/)
+
+## Install and update using pip:
+
+```
+pip install -U cloudstate
+```
+
+## A Simple EventSourced Example:
+
+1. Define your gRPC contract
+
+```
+// This is the public API offered by the shopping cart entity.
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+import "cloudstate/entity_key.proto";
+import "google/api/annotations.proto";
+import "google/api/http.proto";
+
+package com.example.shoppingcart;
+
+message AddLineItem {
+    string user_id = 1 [(.cloudstate.entity_key) = true];
+    string product_id = 2;
+    string name = 3;
+    int32 quantity = 4;
+}
+
+message RemoveLineItem {
+    string user_id = 1 [(.cloudstate.entity_key) = true];
+    string product_id = 2;
+}
+
+message GetShoppingCart {
+    string user_id = 1 [(.cloudstate.entity_key) = true];
+}
+
+message LineItem {
+    string product_id = 1;
+    string name = 2;
+    int32 quantity = 3;
+}
+
+message Cart {
+    repeated LineItem items = 1;
+}
+
+service ShoppingCart {
+    rpc AddItem(AddLineItem) returns (google.protobuf.Empty) {
+        option (google.api.http) = {
+            post: "/cart/{user_id}/items/add",
+            body: "*",
+        };
+    }
+
+    rpc RemoveItem(RemoveLineItem) returns (google.protobuf.Empty) {
+        option (google.api.http).post = "/cart/{user_id}/items/{product_id}/remove";
+    }
+
+    rpc GetCart(GetShoppingCart) returns (Cart) {
+        option (google.api.http) = {
+          get: "/carts/{user_id}",
+          additional_bindings: {
+            get: "/carts/{user_id}/items",
+            response_body: "items"
+          }
+        };
+    }
+}
+
+```
+
+2. Generate Python files
```

Comment:
even present in the README.txt correctly, on pypi the numbering is not rendered properly.

<img width="546" alt="Screenshot 2020-07-10 at 23 04 25" src="https://user-images.githubusercontent.com/12616445/87202906-cecac400-c301-11ea-8727-92bb9926d929.png">

Reply:
the content type still is markdown, perhaps thats the reason numbered sections don't get correct numbering.

https://the-hitchhikers-guide-to-packaging.readthedocs.io/en/latest/creation.html also states:
>The README.txt file should be written in reST

Reply:
I tried a README.txt with all numbered sections to be `1.`, but that did not work:
https://test.pypi.org/project/cloudstate/0.1.1.2/